CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ _None_
 
 - Update the CI image used to build this project to use `xcode-14.1`. [#431]
 
+## 6.2.0
```

Comment:
⚠️  We should not update the version of the CHANGELOG header when adding new entries to the CHANGELOG, because we don't know when we will do a new version of the release-toolkit, which means we also don't know if there will be breaking changes added to the CHANGELOG by future PRs before we do the next release, which would impact the version bump differently according to SemVer.

For tht reason, you should only add your entry under the existing `### New Features` subsection that is under `Trunk` (which is the section for yet-unreleased changes). In that case, replacing the `_None_` line with yours.

It is only when we will decide to do a new release of the release-toolkit and trigger the release process for it (`bundle exec rake new_release`), that we (well, the `rake` the script, really) will determine which version to bump to (according to semver rules and content of the subsections under `Trunk` section at that moment), create the release with appropriate version, and create a new empty `Trunk` section for the next PRs after it.

Reply:
Thanks, it wasn't obvious how we deal with the CHANGELOG entries. But your explanation helped me a lot to understand how the process works. Moved the entry to the correct section in https://github.com/wordpress-mobile/release-toolkit/pull/433/commits/30fd7b4fb27ec951a10e3bb6c37bbee9bd777e9e.